transport/http/constants.go

```diff
@@ -77,8 +91,17 @@ const (
 
 	// An error occurred. The response body contains an application header.
 	ApplicationErrorStatus = "error"
+
+	// AcceptTrue is the true value used for accept headers. Note that any
+	// non-empty value indicates true, but we end up sending this specific
+	// value.
+	AcceptTrue = "true"
 )
 
 // ApplicationHeaderPrefix is the prefix added to application header keys to
 // send them in requests or responses.
 const ApplicationHeaderPrefix = "Rpc-Header-"
+
+func fromAcceptValue(acceptValue string) bool {
+	return len(acceptValue) > 0
```

Comment:
May be better to test exact value. Otherwise, `no` will mean yes. Hyrum’s Law http://www.hyrumslaw.com/

Reply:
Done